src/main/java/principal/Principal.java

```diff
@@ -1,26 +1,26 @@
-package principal;
-
-import control.Control;
-import vista.Vista;
-
-public class Principal {
-	public static void main(String[] args) {
-		try {
-			for (javax.swing.UIManager.LookAndFeelInfo info : javax.swing.UIManager.getInstalledLookAndFeels()) {
-				if ("Nimbus".equals(info.getName())) {
-					javax.swing.UIManager.setLookAndFeel(info.getClassName());
-					break;
-				}
-			}
-		} catch (ClassNotFoundException ex) {
-			java.util.logging.Logger.getLogger(Vista.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
-		} catch (InstantiationException ex) {
-			java.util.logging.Logger.getLogger(Vista.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
-		} catch (IllegalAccessException ex) {
-			java.util.logging.Logger.getLogger(Vista.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
-		} catch (javax.swing.UnsupportedLookAndFeelException ex) {
-			java.util.logging.Logger.getLogger(Vista.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
-		}
-		Control.getInstance().start();
-	}
-}
+package principal;
+
+import control.Control;
+import vista.Vista;
+
+public class Principal {
```

Comment:
faltan imports y os quitáis los `javax.swing.UIManager.LookAndFeelInfo info` y los `java.util.logging.Logger`